Src/Plugins/Tests.AzureDevOpsConfiguration/TestData/ServiceConnectionSecurity/ExpectedOutputs/AZC102_190.serviceconnection.DoNotShareWithAllPipelines.sarif

```diff
@@ -48,7 +48,7 @@
         {
           "ruleId": "AZC102/190",
           "ruleIndex": 0,
-          "level": "note",
+          "level": "error",
```

Comment:
>error [](http://example.com/codeflow?start=20&length=5)

So, yes, this is wrong and it's the bug I pointed out. This rule has a static/dynamic validator but dynamic analysis isn't enabled. So this level *should* have been warning but it is not. I pointed out the code change that I think is responsible for this.

But now we have to ask? why are your tests passing for this case? :) something isn't right, can you figure out what it is?